tests/e2e/input-search.spec.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+
+describe('input search', () => {
+    it('should open the main page by first load', () => {
+        cy.visit('/')
```

Comment:
Cypress only visits the page :D 


![image](https://user-images.githubusercontent.com/48555130/102601339-4e19ba80-4120-11eb-8117-d27c361a795e.png) 
*-- no testing, this is